kura/org.eclipse.kura.rest.position.provider/src/main/java/org/eclipse/kura/rest/position/api/PositionDTO.java

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+ * Copyright (c) 2021, 2022 Eurotech and/or its affiliates and others
+ * 
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ * 
+ * Contributors:
+ *  Eurotech
+ *******************************************************************************/
+package org.eclipse.kura.rest.position.api;
+
+import org.osgi.util.position.Position;
+
+public class PositionDTO {
+
+    private final double longitude;
```

Comment:
org.osgi.util.position.Position Javadoc reports that all of its fields can be null, maybe we should switch to Double to accomodate that and handle that case in the constructor. If we do that, we should also report it in the documentation